### ISSUE TITLE
For DetailList, Narrator/NVDA is not reading the links of the table grid as "link".

### ISSUE THREAD

**Issue by chakradhar2** [OfficeDev/office-ui-fabric-react#6024]
Expected Behaviour 
Narrator/NVDA should read the links of the table grid as "link" along with the name of the data available in the grid cell. 

Example: https://developer.microsoft.com/en-us/fabric#/components/detailslist/variablerowheights

**Comment by natalieethell** [comment]
Hey @chakradhar2 thanks for submitting this issue! I just tested out the behavior using Narrator + Edge, NVDA + Chrome, and NVDA + Edge, and I got what I believe is the behavior you're expecting.

When focusing on a cell in the "key" column, which is a link, the screen readers read out both the name of the key and "link":

**item-1 Lorem ipsum dolor sit, link, ...**

The cells in the "name" column don't actually have any hrefs, so the expected behavior should be that "link" is not read out by the screen reader. Instead, "button" should be read out. This is the behavior I'm experiencing as well.

Is there something else I'm missing here?
